testing/jitterentropy-rngd/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer:
```

Comment:
You don't want to maintain this?

Reply:
I am adding it as something to test for people that report issues with`haveged`, somebody mentioned it on irc the other day I think. If this package does not work better then `haveged` in their systems, we can drop `jitterentropy-rngd` package.